File: example/example_python/function.py
import random
from typing import List, Tuple


def func():
    ans: List[List[int]] = []
    schedule_node: Tuple[int, int, int, int, int]
    for i in range(10):
        operator_id: int = random.randint(0, 10)
        tiling_id: int = random.randint(0, 10)
        node_id_in_operator: int = random.randint(0, 10)
        node_start_time: int = random.randint(0, 10)
        processor_id_for_node: int = random.randint(0, 10)
        schedule_node: Tuple[int, int, int, int, int] = (operator_id, tiling_id, node_id_in_operator, node_start_time, processor_id_for_node)

        ans.append(list(schedule_node))
    print(ans)

if __name__ == '__main__':
    func()

File: example/example_java/Function.java
import java.util.ArrayList;
import java.util.List;
import java.util.Random;

public class Function {
    public static void func() {
        Random random = new Random();
        List<List<Integer>> ans = new ArrayList<>();
        for (int i = 0; i < 10; i++) {
            int operatorId = random.nextInt(11);
            int tilingId = random.nextInt(11);
            int nodeIdInOperator = random.nextInt(11);
            int nodeStartTime = random.nextInt(11);
            int processorIdForNode = random.nextInt(11);
            List<Integer> scheduleNode = new ArrayList<>();
            scheduleNode.add(operatorId);
            scheduleNode.add(tilingId);
            scheduleNode.add(nodeIdInOperator);
            scheduleNode.add(nodeStartTime);
            scheduleNode.add(processorIdForNode);
            ans.add(scheduleNode);
        }
        System.out.println(ans);
    }
}